tests/js/spec/views/organizationIntegrations/integrationListDirectory.spec.jsx

```diff
@@ -0,0 +1,332 @@
+import React from 'react';
+
+import {Client} from 'app/api';
+import {mountWithTheme} from 'sentry-test/enzyme';
+import IntegrationListDirectory from 'app/views/organizationIntegrations/integrationListDirectory';
+
+const mockResponse = mocks => {
+  mocks.forEach(([url, body]) => {
+    return Client.addMockResponse({
+      url,
+      body,
+    });
+  });
+};
+
+describe('IntegrationListDirectory', function() {
+  beforeEach(function() {
+    Client.clearMockResponses();
+  });
+
+  const org = TestStubs.Organization();
```

Comment:
I would look at `sentry-test/initializeOrg` - it sets up org in context (even though it's not needed with these tests)